fvm/evm/types/events.go

```diff
@@ -9,8 +10,6 @@ import (
 const (
 	EventTypeBlockExecuted       flow.EventType = "evm.BlockExecuted"
 	EventTypeTransactionExecuted flow.EventType = "evm.TransactionExecuted"
-	EventTypeFlowTokenDeposit    flow.EventType = "evm.FlowTokenDeposit"
-	EventTypeFlowTokenWithdrawal flow.EventType = "evm.FlowTokenWithdrawal"
```

Comment:
These two events are not needed any more, they are captured as part of tx executed.